content/en/developers/smart-contracts/how-tos/add-to-metamask/index.md

```diff
@@ -62,55 +44,30 @@ Before we get started, you'll need the following:
 
 The process for integrating Filecoin into MetaMask is fairly simple but has some very specific variables that you must copy exactly.
 
-1. Open your browser and open the MetaMask plugin:
-
-    ![MetaMask home.](metamask-home.png)
-
-    If you haven't opened the MetaMask plugin before, you'll be prompted to create a new wallet. Follow the prompts to create a wallet.
-1. If you haven't already, change your view to **Expanded view**:
-
-    ![MetaMask in expanded view mode.](expand-view.png)
-
-1. Click the user circle and select **Settings**:
-
-    ![MetaMask user dropdown.](select-settings.png)
-
+1. Open your browser and open the MetaMask plugin. If you haven't opened the MetaMask plugin before, you'll be prompted to create a new wallet. Follow the prompts to create a wallet.
+1. If you haven't already, change your view to **Expanded view**.
+1. Click the user circle and select **Settings**
 1. Select **Networks**.
-
-    ![MetaMask menu with Networks highlighted.](networks-tab.png)
-
-1. Click **Add a network**:
-
-    ![](add-a-network.png)
-
-1. Scroll down and click **Add a network manually**:
-
-    ![](add-a-network-manually.png)
-
+1. Click **Add a network**.
+1. Scroll down and click **Add a network manually**.
 1. Enter the following information into the fields:
 
     | Field | Value |
     | --- | --- |
-    | Network name | `Filecoin Wallaby` |
-    | New RPC URL | `https://wallaby.node.glif.io/rpc/v0` |
+    | Network name | `Filecoin Hyperspace` |
+    | New RPC URL | `https://api.hyperspace.node.glif.io/rpc/v0` |
```

Comment:
have talked to glif - confirmed will only serve eth api on v1 after todays patch, so lets update the endpoint